edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Kubernetes.Test/KubernetesDeploymentMapperTest.cs

```diff
@@ -40,6 +40,16 @@ public class KubernetesDeploymentMapperTest
             }
         };
 
+        static readonly HostConfig HostIpcModeHostConfig = new HostConfig
+        {
+            IpcMode = "host"
+        };
+
+        static readonly HostConfig PrivateIpcModeHostConfig = new HostConfig
+        {
+            IpcMode = "private"
+        };
```

Comment:
does **shareable** IPC mode make sense for k8s scenario?

Reply:
For unit tests, I really only needed IpcMode not set, set to host, and not set to host, since those are the only cases relevant to k8s API settings.